src/storagesync/README.md

```diff
@@ -0,0 +1,202 @@
+==========================================
+# Azure CLI Storage Sync Extension #
+This is a extension for StorageSync features.
+
+### How to use ###
+Install this extension using the below CLI command
+```
+az extension add --name storagesync
+```
+
+### Included Features
+#### Manage storage sync service:
+
+##### Create a new storage sync service.
+
+```
+az storagesync storage-sync-service create \
+    --resource-group rg \
+    --name storage_sync_service_name \ 
+    --location westus \
+    --tags key1=value1
+```
+
+##### Delete a given storage sync service.
+```
+az storagesync storage-sync-service delete \
+    --resource-group rg \
+    --name storage_sync_service_name
+```
+
+##### Show the properties for a given storage sync service.
+```
+az storagesync storage-sync-service show \
+    --resource-group rg \
+    --name storage_sync_service_name
+```
+
+##### List all storage sync services in a resource group or a subscription.
+```
+az storagesync storage-sync-service list
+```
+```
+az storagesync storage-sync-service list \
+    --resource-group rg
+```
+
+#### Manage sync group:
+
+##### Create a new sync group.
+```
+az storagesync sync-group create \
+    --resource-group rg \
+    --name sync_group_name \
+    --storage-sync-service-name storage-sync-service-name
+```
+
+##### Delete a given sync group.
+```
+az storagesync sync-group delete \
+    --resource-group rg \
+    --name sync_group_name \
+    --storage-sync-service-name storage-sync-service-name
+```
+
+##### Show the properties for a given sync group.
+```
+az storagesync sync-group show \
+    --resource-group rg \
+    --name sync_group_name \
+    --storage-sync-service-name storage-sync-service-name
+```
+
+##### List all sync groups in a storage sync service.
+```
+az storagesync sync-group list \
+    --resource-group rg \
+    --storage-sync-service-name storage-sync-service-name
+```
+
+#### Manage cloud endpoint.
+
+##### Create a new cloud endpoint.
+```
+az storagesync cloud-endpoint create \
+    --resource-group rg \
+    --name cloud-endpoint-name \
+    --storage-sync-service-name storage-sync-service-name \
```

Comment:
`storage-sync-service` to support both name and id?

Reply:
no, only support name

---
In reply to: [392648605](https://github.com/Azure/azure-cli-extensions/pull/1389#discussion_r392648605) [](ancestors = 392648605)

Reply:
support both now

---
In reply to: [392866324](https://github.com/Azure/azure-cli-extensions/pull/1389#discussion_r392866324) [](ancestors = 392866324)